manual/working/scalaGuide/main/tests/ScalaTestingWithDatabases.md

```diff
@@ -1,110 +1,218 @@
+-->
+[[ScalaTest|ScalaFunctionalTestingWithScalaTest]] や [[specs2|ScalaFunctionalTestingWithSpecs2]] を使って、データベースを含むアプリケーション全体を起動してデータベースアクセスコードをテストするような機能テストを書くことも可能ではありますが、アプリケーションのたった一部分をテストするために多くのコンポーネントを立ち上げるという複雑性により、アプリケーション全体を立ち上げるのは通常好ましくありません。
 
+<!--
 Play provides a number of utilities for helping to test database access code that allow it to be tested with a database but in isolation from the rest of your app.  These utilities can easily be used with either ScalaTest or specs2, and can make your database tests much closer to lightweight and fast running unit tests than heavy weight and slow functional tests.
+-->
+Play はデータベースアクセスコードをテストする補助となる多くのユーティリティを提供しており、他の部分から独立してデータベースのみをテストできるようにしています。このようなユーティリティは ScalaTest や specs2 から容易に使用でき、データベースのテストを重くて遅い機能テストからより軽量で高速なものとすることができます。
 
+<!--
 ## Using a database
+-->
+## データベースを使う
 
+<!--
 To connect to a database, at a minimum, you just need database driver name and the url of the database, using the [`Databases`](api/scala/play/api/db/Databases$.html) companion object.  For example, to connect to MySQL, you might use the following:
+-->
+データベースに接続するには、最低限、データベースドライバ名とデータベースの URL が必要で、 [`Databases`](api/scala/play/api/db/Databases$.html) コンパニオンオブジェクトを使用します。例えば、 MySQL に接続するには以下のようにします。
 
 @[database](code/database/ScalaTestingWithDatabases.scala)
 
+<!--
 This will create a database connection pool for the MySQL `test` database running on `localhost`, with the name `default`.  The name of the database is only used internally by Play, for example, by other features such as evolutions, to load resources associated with that database.
+-->
+これによって、 `default` という名前の `localhost` で実行される  MySQL の `test` データベースへのコネクションプールが生成されます。データベースの名前は、例えば evolutions などの他の機能ではデータベースに関連づけられたリソースをロードするなど、Play の内部でのみ使用されるものです。
 
+<!--
 You may want to specify other configuration for the database, including a custom name, or configuration properties such as usernames, passwords and the various connection pool configuration items that Play supports, by supplying a custom name parameter and/or a custom config parameter:
+-->
+独自の name や 、 username や password、Play がサポートする様々なコネクションプール設定アイテムなどの設定プロパティなど、データベースについて他にも設定したいときは、カスタム name パラメータやカスタム config パラメータを与えることができます。
 
 @[full-config](code/database/ScalaTestingWithDatabases.scala)
 
+<!--
 After using a database, since the database is typically backed by a connection pool that holds open connections and may also have running threads, you need to shut it down.  This is done by calling the `shutdown` method:
+-->
+データベースの使用後は、データベースはよく、コネクションが開いている、ことによるとスレッドが実行中の可能性すらあるコネクションプールによって戻されることがあるので、閉じる必要があります。これは `shutdown` メソッドで行うことができます。
```

Comment:
`since the database is typically backed` の `back` は意訳したいですが、いい訳が思い付かない...

- [backの意味・用例｜英辞郎 on the WEB：アルク](http://eow.alc.co.jp/search?q=back)

コネクションが開きっぱなしになっているネガティブな状態のことを `backed` と言っている気がするのですが...